pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -86,13 +86,21 @@ func FindTestTarget(labels []configsections.Label, target *configsections.TestTa
 			target.Operators = append(target.Operators, buildOperatorFromCSVResource(&csv))
 		}
 	}
-	dps := FindTestDeploymentsByLabel(labels, target)
-	for _, dp := range dps {
-		if ns[dp.Namespace] {
-			target.DeploymentsUnderTest = append(target.DeploymentsUnderTest, dp)
+	dps := FindTestPodSetsByLabel(labels, target, string(configsections.Deployment))
+	target.DeploymentsUnderTest = appendPodsets(dps, ns)
+	stateFullSet := FindTestPodSetsByLabel(labels, target, string(configsections.StateFullSet))
```

Comment:
Would it be possible to replace all occurrences of statefullset for statefulset? I know it's a bit late in the review cycle, but it would reduce confusion in the future.